src/middlewared/middlewared/plugins/interface/configure.py

```diff
@@ -170,10 +175,17 @@ def configure(self, data, aliases, wait_dhcp=False, options=None):
                 iface.carp_config = [netif.CarpConfig(carp_vhid, advskew=advskew, key=carp_pass.encode())]
         else:
             if vrrp_vip:
+                if not self.middleware.call_sync('service.started', 'keepalived'):
+                    self.middleware.call_sync('service.start', 'keepalived')
+                else:
+                    self.middleware.call_sync('service.reload', 'keepalived')
```

Comment:
is that going to reload/start keepalived for every interface?

Reply:
It shouldn't, only reloads the service if there is a change to the `vrrp_vip` attribute.